### PR TITLE
enable list-based execution strategy for bazel rbe build

### PR DESCRIPTION
**Pull request by ishikhman**
set --incompatible_list_based_execution_strategy_selection=true

Fixing bazelbuild/bazel#8372

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -11,11 +11,8 @@ build:remote --extra_execution_platforms=//:rbe_highcpu_platform
 build:remote --host_platform=@bazel_rbe_toolchains//configs/bazel_0.25.0/bazel-ubuntu1804:default_platform
 build:remote --platforms=@bazel_rbe_toolchains//configs/bazel_0.25.0/bazel-ubuntu1804:default_platform
 
-build:remote --spawn_strategy=remote
-build:remote --strategy=Javac=remote
-build:remote --strategy=Closure=remote
-build:remote --strategy=Genrule=remote
-build:remote --define=EXECUTOR=remote
```
Comment (buchgr) [inline .bazelrc]: we still need the EXECUTOR=remote

Comment (ishikhman) [reply .bazelrc]: Oh, good catch!

**Comment by ishikhman** [comment]
> Should `build:remote --strategy=DexBuilder=remote` also be removed?

removed. It should work without any strategy flags. Theoretically :)

**Comment by buchgr** [comment]
LGTM

**Pull request closed without merging**